src/views/Layout.jsx

```diff
@@ -37,6 +37,60 @@ export function Layout() {
 						</NavLink>
 					</div>
 				</nav>
+				<footer className="Layout-footer">
+					<p className="Layout-footer-attribution">
+						Crafted with care{' '}
+						<span role="img" aria-label="heart">
+							❤️
+						</span>
+						by{' '}
```

Comment:
Super nit: perhaps a space after the emoji?

Reply:
Good catch. Will update!